docker/edge/http/script.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+#apt-get install jq -y
+
+#Read the Keyrock IP, Application name, redirect_uri and application URL from user
+echo Enter the Keyrock IDM IP:
+read IP
+IP_ADDRESS=`echo $IP | tr -d '\n'`
+echo $IP | tr -d '\n'|od -c
+echo IDM IP is $IP_ADDRESS
+echo -----------------------------
+echo Enter name of Application:
+read name
+appName=`echo $name | tr -d '\n'`
+echo Name of application is $appName
+echo -----------------------------
+echo Enter redirect_uri/Callback_uri:
+read callback
+callbackUrl=`echo $callback | tr -d '\n'`
+echo -----------------------------
+
+echo Enter application url:
+read Uri
+appUri=`echo $Uri | tr -d '\n'`
+echo -----------------------------
+
+#Generate API token, It will return X-Subject-Token taht will further use for application register
```

Comment:
[Major] if curl is not present in the linux then it will fail without any error. so better to check whether curl is installed or not, if not installed then need to install it first.

Reply:
Added check before using curl command